docs/embedding/signed-embedding-parameters.md

```diff
@@ -54,22 +55,23 @@ If the original dashboard's filter widget accepts multiple values (i.e., it's a
 your_embedding_url?breakfast=Scrambled_eggs&breakfast=Bacon
 ```
 
-## Pre-filtering data in a signed embed
+## Restricting data in a signed embed
 
-If you want to display filtered data in an embedded dashboard or SQL question, and _prevent_ people from viewing or changing that filter:
+If you want to restrict the data that's displayed in an embedded dashboard or SQL question, you can set up a locked parameter. A locked parameter filters your data _before_ the results are displayed to the end user.
+
+You can use locked parameters to display filtered data based on attributes captured by your web server (such as a user's login). For more examples, see the [reference apps repo](https://github.com/metabase/embedding-reference-apps).
 
 1. Go to your dashboard or SQL question. Make sure you've set up a [dashboard filter](../dashboards/filters.md) or [SQL variable](../questions/native-editor/sql-parameters.md).
 2. Click on the **sharing icon** > **Embed this item in an application**.
 3. Under **Parameters**, you'll find the names of your dashboard filters or SQL variables.
 4. Select **Locked** for each parameter that should pre-filter your data.
 5. Add values for the filter under **Preview locked parameters**.
 6. Click **Publish** to save your changes.
+7. Add or update the code on your server so that it matches the code generated by Metabase (Code > top block).
```

Comment:
```suggestion
7. Add or update the code on your server to match the code generated by Metabase (Code > top block).
```

Can you explain what you mean by "Code > top block"?

Reply:
Yeah, it looks like this:

<img width="1140" alt="publish-dashboard" src="https://user-images.githubusercontent.com/22361741/197009585-c231e118-bf17-433d-96dc-2c45dc438b44.png">